<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1271</width>
    <height>710</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1271</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TEMU</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/pics/meidusha.ico</normaloff>:/pics/meidusha.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>640</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="cmd">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>891</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>640</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>891</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>670</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>RegInfo</string>
     </attribute>
     <widget class="QTextBrowser" name="regInfo">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>341</width>
        <height>650</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>CP0Info</string>
     </attribute>
     <widget class="QTextBrowser" name="cp0Info">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>341</width>
        <height>650</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopen">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionopen"/>
   </widget>
   <addaction name="menuopen"/>
  </widget>
  <action name="actionopen">
   <property name="icon">
    <iconset>
     <normaloff>../../../图片/9c22429f235e4f98808b6ad7dc8ccd80.jpg</normaloff>../../../图片/9c22429f235e4f98808b6ad7dc8ccd80.jpg</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
